docs/resources/dataarts_factory_script.md

```diff
@@ -0,0 +1,84 @@
+---
+subcategory: "DataArts Studio"
+---
+
+# huaweicloud_dataarts_factory_script
+
+Manages DataArts factory script resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "workspace_id" {}
+variable "name" {}
+variable "connection_name" {}
+
+resource "huaweicloud_dataarts_factory_script" "test" {
+  workspace_id    = var.workspace_id
+  name            = var.name
+  type            = "DLISQL"
+  content         = "#content"
+  connection_name = var.connection_name
+  script          = "/"
+  queue_name      = "default"
+  description     = "test"
+  configuration   = {
+    "spark.sql.files.maxRecordsPerFile" = "1"
+    "dli.sql.job.timeout"               = "1"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to manage the script.
+  Changing this creates a new script.
+
+* `workspace_id` - (Required, String, ForceNew) Specifies the workspace ID which the script in.
+  Changing this creates a new script
+
+* `name` - (Required, String, ForceNew) Specifies the script name.
```

Comment:
please add more details about `name`, `type` and `content`, you can get them from API docs in English version.